v03_pipeline/lib/tasks/write_metadata_for_run.py

```diff
@@ -75,15 +75,20 @@ def run(self) -> None:
             'callsets': self.callset_paths,
             'run_id': self.run_id,
             'sample_type': self.sample_type.value,
-            'families': {},
-            'families_failed_missing_samples': {},
-            'families_failed_relatedness_check': {},
-            'families_failed_sex_check': {},
+            'family_samples': {},
+            'family_samples_failed_missing_samples': {},
```

Comment:
to clarify, is `family_samples` only the families and samples that succeed or is it all families and samples in the run, and then the failed ones are listed in the failed directory

Reply:
`family_samples` is only the families and samples that succeed.